server/gendb/dynamodb.go.tmpl

```diff
@@ -39,6 +39,8 @@ type Config struct {
   {{- end }}
 }
 
+const maxDynamoDBBatchItems = 25
```

Comment:
Can we add a comment here that this limitation is set by aws. Linking to https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html